src/Okta.Sdk.UnitTests/OktaClientShould.cs

```diff
@@ -344,5 +342,12 @@ public async Task RetrieveNestedListOfObjects()
             statements.First().Type.Should().Be("EXPRESSION");
             statements.First().Name.Should().Be("Attribute");
         }
+
+        [Fact]
+        public void RegisterPkixCertPayloadHandlerOnConstruction()
+        {
+            _ = new OktaClient();
```

Comment:
Is this necessary?

Reply:
The underscore is a recommendation from Visual Studio (https://docs.microsoft.com/en-us/dotnet/csharp/discards).  The test asserts that construction of the client registers the payload handler.

Reply:
Oh I see! you call the constructor to register the payloads.